docs/products/kafka/howto/configure-topic-tiered-storage.md

```diff
@@ -1,146 +1,127 @@
 ---
 title: Enable and configure tiered storage for topics
-early: true
 ---
 
-Aiven for Apache Kafka® allows you to configure tiered storage
-and set retention policies for individual topics. Learn how to configure
-tiered storage for individual topics and set local retention policies
-step by step.
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
 
-## Prerequisite
 
--   [Tiered storage enabled for the Aiven for Apache Kafka service](/docs/products/kafka/howto/enable-kafka-tiered-storage).
+Aiven for Apache Kafka® allows you to configure tiered storage and set retention policies for individual topics.
 
-## Configure tiered storage for topics via Aiven Console
+## Prerequisite
 
-1.  Access [Aiven console](https://console.aiven.io/), select your
-    project, and select your Aiven for Apache Kafka service.
-1.  From the left sidebar, select **Topics**.
-1.  You have the option to either add a new topic with tiered
-    storage configuration or modify an existing topic to use tiered
-    storage.
+- [Tiered storage enabled for the Aiven for Apache Kafka service](/docs/products/kafka/howto/enable-kafka-tiered-storage).
```

Comment:
Consider removing the bullet point and including the prerequisite not as a list item (since there are no more items to be listed).